spec/system/support_interface/view_course_type_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'View course choice' do
+  include DfESignInHelpers
+
+  before do
+    given_i_am_a_support_user
+  end
+
+  scenario 'View postgraduate course choice' do
```

Comment:
Do we need a full blown feature test for this? Can't we have a lightweight component spec?

`course_option_details_component_spec.rb`?

Reply:
Due to importance of the feature I wanna keep the system/feature test. If is not there it could cause confusion on what course we are dealing with.

I can add the component if you want, what do you think?

Reply:
I added the test on c9acc10758

Reply:
I don't think there needs to be a file with the name `view_course_type_spec.rb`.
It's a full file for testing one row in a table.

Reply:
I still think a feature test should be there for this due to importance. Removed on 87e7677ed3

Reply:
I think there should be a related component spec per component. That's what we have on Publish

Reply:
> I think there should be a related component spec per component. That's what we have on Publish

Yes, I agree. The support is full of without corresponding specs.